### DISCUSSION TITLE
Github 2FA recovery not responding after a week

### DISCUSSION THREAD

**Discussion by ziidonato** [community/community#138791] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I lost access to my password manager and thus my 2FA code and recovery code. Luckily I'm still logged in on my laptop and desktop, so I can post this.

Github support says they'll respond in 1-3 business days for this type of request. It's been 5. I have done everything I was supposed to do. I went to forgot password, clicked to reset it, did the 2FA recovery steps, verified the SSH key, and now it just says they'll respond in 1-3 business days.

It's been a week, and I still don't have any responses or anything indicating that there are steps being taken to fix this. I actually just tried again, and it let me take the steps to verify it again. Does it mean they denied it the first time?

Why is this so difficult? Why do they say 1-3 business days then do this? It's infuriating to be given a timeframe and then have it not honored at all.

**Comment by ebndev** [comment]
Hi @ziidonato,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
